tests/dummy/app/templates/snippets/datepicker-1.hbs

```diff
@@ -1,6 +1,6 @@
 {{#basic-dropdown as |dropdown|}}
   <input type="text"
-    id="ember-basic-dropdown-trigger-{{dropdown.uniqueId}}"
+    data-ebd-id="{{dropdown.uniqueId}}-trigger"
```

Comment:
`dropdown.uniqueId` is public API. However the fact that `data-ebd-id` is a magical attribute I use to wire things toguether, while I don't intent to change it, it's pretty hard to document as public. It's basically an implementation detail I don't plan to change.